src/models/table.ts

```diff
@@ -0,0 +1,4 @@
+export interface TableState {
```

Comment:
Interface should use for case Inheritance and abstract thing.
The good way to use should be `type`

```typescript
type Table = {
page: number;
size: number;
}
```